repl/src/main/scala/ammonite/repl/interp/Evaluator.scala

```diff
@@ -170,25 +172,28 @@ object Evaluator{
         // Exhaust the printer iterator now, before exiting the `Catching`
         // block, so any exceptions thrown get properly caught and handled
 
-        val iter = evalMain(cls).asInstanceOf[Iterator[String]]
+        val iter = cls.getDeclaredMethod("$main").invoke(null).asInstanceOf[Iterator[String]]
```

Comment:
I don't know why but it is necessary, without that weird `BoxedUnit`error starts coming for repl cmds